Libs/MRML/Widgets/qMRMLWidget.cxx

```diff
@@ -94,16 +94,6 @@ void qMRMLWidget::preInitializeApplication()
   if (openGLProfileStr.isEmpty() || openGLProfileStr == "default")
```

Comment:
Since you have remove the exception for win32, and the format is initialized to default, wouldn't it make sense to remove this if block?

Reply:
True. Or we could add a final "else" branch and log an error there, telling that an invalid profile option was set. For example, it would warn the user when he set "none" instead of the valid "no" option.